src/System.Text.Json/tests/Serialization/CustomConverterTests.BadConverters.cs

```diff
@@ -76,8 +77,102 @@ private class PocoWithNullConverter
         [Fact]
         public static void AttributeCreateConverterFail()
         {
-            Assert.Throws<InvalidOperationException>(() => JsonSerializer.Serialize(new PocoWithNullConverter()));
-            Assert.Throws<InvalidOperationException>(() => JsonSerializer.Deserialize<PocoWithNullConverter>("{}"));
+            try
```

Comment:
I don't think changing to try/catch is necessary. We can keep using Assert.Throws, which returns the exception.
```C#
InvalidOperationException ex = Assert.Throws<InvalidOperationException>(() => JsonSerializer.Serialize(new PocoWithNullConverter()));
Assert.Contains("'System.Text.Json.Serialization.Tests.CustomConverterTests+PocoWithNullConverter.MyInt'", ex.Message);
```

Reply:
I never realized that. I'll clean this file up.